### PR TITLE
Bump Go version to 1.9.3.

### PR DESCRIPTION
**Pull request by AlekSi**
/cc @BanzaiMan

### CODE REVIEW

**Comment by BanzaiMan** [comment]
Spec needs fixing.

**Comment by AlekSi** [comment]
Looks like not a spec, but something else should be fixed: `1.9.x` should resolve into `1.9.3`, not `1.9.1`.

**Comment by AlekSi** [comment]
I guess it's gimme's `.testdata`. But now I'm totally confused.

**Comment by BanzaiMan** [comment]
Well, the file is here: https://github.com/travis-ci/gimme/blob/master/.testdata/sample-binary-linux

@meatballhat Could you shine a light on how we are supposed to update the default Go version?

**Comment by AlekSi** [comment]
I already sent https://github.com/travis-ci/gimme/pull/120. But the fact that travis-build depends on gimme's _testdata_ sounds very strange for me. See also https://github.com/travis-ci/gimme/issues/110.

**Comment by philpennock** [comment]
travis-ci/gimme change updating testdata approved and merged.  Yinz should be good to trigger a rebuild of this one now, to confirm it works.

**Comment by AlekSi** [comment]
I rebased my branch to trigger a build, but it still fails. And it shows old values.

File: spec/build/rake_tasks_spec.rb [hunk 1]
```diff
@@ -210,16 +210,16 @@ def releases_response(*versions)
...
       '1.4.0' => '1.4.0',
       '1.4.x' => '1.4.0',
-      '1.9.1' => '1.9.1',
-      '1.9.x' => '1.9.1',
-      '1.x' => '1.9.1',
-      '1.x.x' => '1.9.1',
+      '1.9.3' => '1.9.3',
+      '1.9.x' => '1.9.3',
+      '1.x' => '1.9.3',
+      '1.x.x' => '1.9.3',
```
Comment (meatballhat) [inline spec/build/rake_tasks_spec.rb]: The spec that is failing in this file is due to these values having been changed.  The expectation here is driven by stubbed test data and _should not_ be considered a part of the production code go version aliases 💖 

Said another way: revert the changes to this file and the specs should pass.

**Comment by AlekSi** [comment]
Closed in favor of #1307.

**Pull request closed without merging**